packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -15,13 +15,49 @@ export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
           ...newObj,
           ...nextObject,
         }
+
+        if (key === 'custom') {
```

Comment:
Ignore `custom` as this object is dynamic.

Reply:
I would move this to a constant, maybe?